package.json

```diff
@@ -6052,35 +6052,35 @@
 				{
 					"id": "gitlens.views.repositories:gitlens",
 					"name": "Repositories",
-					"when": "config.gitlens.views.repositories.enabled && config.gitlens.views.repositories.location == gitlens",
+					"when": "config.git.enabled && gitOpenRepositoryCount != 0 && config.gitlens.views.repositories.enabled && config.gitlens.views.repositories.location == gitlens",
```

Comment:
@eamodio I've followed the approach outlined here (https://github.com/microsoft/vscode/issues/49145#issuecomment-392563659) where if no views meet the `when` condition, then the sidebar icon/button will not be displayed. I'm not sure if there is another/cleaner approach to do so 🙂